google-load-balancer-service.js

```diff
@@ -0,0 +1,289 @@
+const GCCompute = require("@google-cloud/compute");
```

Comment:
I would name it `google-load-balancer-service-adapter` to emphasize it's not any instance, but a code which adapts usage of an existing service (GCP Load balancer).